docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,810 @@
+
+Output:
+```zsh
+Running on subnet
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114
+Safe transaction hash: 0x5134dc35909ff592c55a64c1a5947dd4844b1bca2a45df68ed9c3019133bf44d
+```
+
+#### Sign with Two Owners
+
+Follow the same workflow from before to sign a proposal.
+
+```zsh
+yarn safe sign-proposal 0x5134dc35909ff592c55a64c1a5947dd4844b1bca2a45df68ed9c3019133bf44d
+```
+
+Output:
+```zsh
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 with 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+Signature: 0x636ba2a89023b1e81032a43dd1172743f7916e31647eb87ec95c541c091ebf1873605d39d8039431a7dceeeab691e48b96b50f93e91acde5e67295e9f051e7031f
+```
+
+By default, Hardhat uses _account 0_ to sign transactions. Since we've imported another private key and added it to our _accounts_ parameter in `hardhat.config.ts` we can now specify which account we want to sign with by adding the flag `--signer` to our `sign-proposal` task 
+
+```zsh
+yarn safe sign-proposal 0x5134dc35909ff592c55a64c1a5947dd4844b1bca2a45df68ed9c3019133bf44d --signer 1
+```
+
+Output:
+```zsh
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 with 0x82DdaF3f1fcd3c18F5664cD7fb12bD8C38D5d4ba
+Signature: 0x11d7e983417280bdf1c55da51359eb06262f0feadad1c6ebdf497a6e6db92c5e506536c1c2b6bd3ef726d163c710d5adcbe787a2440be5ad79cac52e950407b21f
+```
+
+#### Submit
+Now that both owners have signed the proposal, the `threshold` requirement has been met and we can now submit the proposal. 
+
+```zsh
+yarn safe submit-proposal 0x5134dc35909ff592c55a64c1a5947dd4844b1bca2a45df68ed9c3019133bf44d
+```
+
+Output:
+```zsh
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 with 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+Ethereum transaction hash: 0x074d823b8d111af9e87d0e4374e3a5382a4de9952df4f49db5ee4b52f945760b
+```
+
+Now lets check the balances of the Safe and EOA addresses using curl.
+
+**Safe balance(0)**
+
+```zsh
+curl -X POST localhost:49435/ext/bc/2Ek1MWR7jiEJr3o9tuJAH79JkuERzKqQDcR2s6R2e5Dyz54Wit/rpc -H "Content-Type: application/json" --data '
+{
+  "jsonrpc": "2.0",
+  "method": "eth_getBalance",
+  "params": ["0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114", "latest"],
+  "id": 1
+}
+```
+Output:
+```zsh
+{"jsonrpc":"2.0","id":1,"result":"0x0"}
+```
+
+**EOA Balance(1,000)**
+
+```zsh
+curl -X POST "http://127.0.0.1:17773/ext/bc/8ttPWTKt2FEs256fJkV2Yj5nJS1JPSfhN2ghAr8aboZWF2gXF/rpc" -H "Content-Type: application/json" --data '
+{
+  "jsonrpc": "2.0",
+  "method": "eth_getBalance",
+  "params": ["0x2d1d87fF3Ea2ba6E0576bCA4310fC057972F2559", "latest"],
+  "id": 1
+}
+```
+Output
+```zsh
+{"jsonrpc":"2.0","id":1,"result":"0x3635c9adc5dea00000"}
+```
+
+And there you have it! We've transferred 1000 _LEVM_ from our Safe to address, `0x2d1d87fF3Ea2ba6E0576bCA4310fC057972F2559`.
```

Comment:
Please describe why  0x3635c9adc5dea00000 is 1000 LEVM.

Reply:
https://github.com/ava-labs/avalanche-docs/pull/917/files#diff-91a1c0545f4a63e1c90f080a247cf8325acdd1d47d3cb42120813d1b35fd7ca5R419-R428